src/sentry/search/django/backend.py

```diff
@@ -21,6 +21,41 @@
 
 
 class DjangoSearchBackend(SearchBackend):
+    def _tags_to_filter(self, project, tags):
+        # Django doesnt support union, so we limit results and try to find
+        # reasonable matches
+        from sentry.models import GroupTagValue
+
+        # ANY matches should come last since they're the least specific and
+        # will provide the largest range of matches
+        tag_lookups = sorted(tags.iteritems(), key=lambda x: x != ANY)
+
+        # get initial matches to start the filter
+        matches = None
```

Comment:
Not sure what's happening here, because this value never gets reassigned to something else, but down on L51 you check for it, which will never be truthy. So not entirely sure the intent here.